src/CMakeLists.txt

```diff
@@ -201,6 +201,10 @@ if (NOT MSVC)
         target_link_libraries( exiv2lib PRIVATE ${CMAKE_DL_LIBS}) # -ldl = dynamic loader used by src/version.cpp
     endif()
 
+    if ( CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" )
+        target_link_libraries( exiv2lib PRIVATE -lprocstat)
```

Comment:
We should probably check first if the library is available in the system with `find_library()`. Or is this a library guaranteed to be on FreeBSD?

Reply:
Testing always make sense.  I think libprocstat is part of the "core" as I don't remember installing with with pkg.    And the command `$ pkg list` enumerates 27488 files.  They are all in /usr/local/ and doesn't include anything about procstat.

There's no Find*proc*.cmake, so we can't search for the include file <libprocstat.h`>  If we're in FreeBSD and we don't have the family of libprocstat resources, we can't build!